roles/sap_ha_set_hana/README.md

```diff
@@ -5,117 +5,129 @@ Ansible role for SAP HANA High Availability Setup
-    sap_ha_set_hana_load_balancer_ip: "10.1.1.1"
-    sap_ha_set_hana_site1: "hana01"
-    sap_ha_set_hana_site2: "hana02"
-    ```
-
-- Run 2 - part 1 - secondary:
-    ```bash
-    ansible-playbook --connection=local --limit localhost -i "localhost," sap-hana-ha.yml -e "@input_file2.yml"
-    ```
-    ```yaml
-    # input_file2.yml contents
-    sap_ha_set_hana_type: "az"
-    sap_ha_set_hana_role: "secondary"
-    sap_ha_set_hana_part: "1"
-    sap_ha_set_hana_sid: "HDB"
-    sap_ha_set_hana_instance_number: "00"
-    sap_ha_set_hana_load_balancer_ip: "10.1.1.1"
-    sap_ha_set_hana_site1: "hana01"
-    sap_ha_set_hana_site2: "hana02"
-    ```
-
-- Run 3 - part 2 - primary:
-    ```bash
-    ansible-playbook --connection=local --limit localhost -i "localhost," sap-hana-ha.yml -e "@input_file3.yml"
-    ```
-    ```yaml
-    # input_file3.yml contents
-    sap_ha_set_hana_type: "az"
-    sap_ha_set_hana_role: "primary"
-    sap_ha_set_hana_part: "2"
-    sap_ha_set_hana_sid: "HDB"
-    sap_ha_set_hana_instance_number: "00"
-    sap_ha_set_hana_load_balancer_ip: "10.1.1.1"
-    sap_ha_set_hana_site1: "hana01"
-    sap_ha_set_hana_site2: "hana02"
-    ```
-
-- Run 4 - part 2 - secondary:
-    ```bash
-    ansible-playbook --connection=local --limit localhost -i "localhost," sap-hana-ha.yml -e "@input_file4.yml"
-    ```
-    ```yaml
-    # input_file4.yml contents
-    sap_ha_set_hana_type: "az"
-    sap_ha_set_hana_role: "secondary"
-    sap_ha_set_hana_part: "2"
-    sap_ha_set_hana_sid: "HDB"
-    sap_ha_set_hana_instance_number: "00"
-    sap_ha_set_hana_load_balancer_ip: "10.1.1.1"
-    sap_ha_set_hana_site1: "hana01"
-    sap_ha_set_hana_site2: "hana02"
-    ```
-
-## Variables / Inputs
-
-| **Variable**                          | **Info**                                  | **Default** | **Required** |
-| :---                                  | :---                                      | :---        | :---         |
-| sap_ha_set_hana_type                  | Cloud type - `az` for Azure               | <none>      | yes          |
-| sap_ha_set_hana_role                  | `primary` or `secondary`                  | <none>      | yes          |
-| sap_ha_set_hana_part                  | `1` or `2`                                | <none>      | yes          |
-| sap_ha_set_hana_sid                   | SID of the SAP HANA system                | <none>      | yes          |
-| sap_ha_set_hana_instance_number       | Instance number of the SAP HANA system    | <none>      | yes          |
-| sap_ha_set_hana_load_balancer_ip      | IP address of the load balancer           | <none>      | yes          |
-| sap_ha_set_hana_site1                 | IP address of the load balancer           | <none>      | yes          |
-| sap_ha_set_hana_site2                 | IP address of the load balancer           | <none>      | yes          |
+- This Ansible role can be used very flexible. It runs the right tasks in the right direction on 
```

Comment:
Shouldn't need a bullet point, and the line break in the sentence would break the bullet point too

Reply:
Resolved by 3218e06